handlers/dice.py

```diff
@@ -0,0 +1,41 @@
+import random
+from array import array
+
+class Dice():
+  def __init__(self):
+    #warning, not true randomness
+    random.seed()

```

Comment:
would it work if you moved this to the main method of mitybot? is that better design?